docs/inventory.md

```diff
@@ -536,4 +536,52 @@ even in parallel by the user by running:
 $ ansible-playbook -i inventory.yml playbook playbooks/deploy_registry.yml -e populate_operator_catalog=True
 ```
 
-If you wish to populate the registry as part of deploying the pre-requistes you can add `populate_operator_catalog: true` to the `registry_host`
\ No newline at end of file
+If you wish to populate the registry as part of deploying the pre-requistes you can add `populate_operator_catalog: true` to the `registry_host`
+# Automated non-node DNS entries.
+
+## DNS Entries for Bastion, Services and VM_Hosts.
+
+When using the crucible provided DNS the automation will add entries for the bastion, service and vm hosts. We will use the value in `ansible_fqdn` except in the cases where, the `registry_host` has `registry_fqdn` defined, or when `vm_hosts` have `sushy_fqdn` defined. The DNS entries will only be defined if the `ansible_host` is an IP address otherwise it will be skipped. If you wish to force the automation to skip a host you can add `dns_skip_record: true` to the host definition. 
+
+## DNS Entries for BMCs
+
+It is posisble for to make crucible create DNS entries for your nodes BMCs. 
+You must using the crucible provided DNS. 
+Then you are required to add entries for `bmc_ip`. The address given will be `{{ inventory_hostname }}.dns_bmc_address_suffix` which defaults to `{{ inventory_hostname}}-bmc.infra.{{ base_dns_domain}}`. For the host below `super1` below will be `"super1-bmc.infra.example.com"`. 
+This can be useful when working with proxies as you can add `*.infra.example.com` to your no_proxy setting.
+
+```yaml
+all:
+  vars:
+    base_dns_domain: example.com
+  ...
+    masters:
+      vars:
+        role: master
+        vendor: SuperMicro
+      hosts:
+        super1:
+          ansible_host: 192.168.10.11
+          bmc_ip: 172.30.10.1
+          # ...
+        super2:
+          ansible_host: 192.168.10.12
+          bmc_ip: 172.30.10.2
+          # ...
+        super3:
+          ansible_host: 192.168.10.13
+          bmc_ip: 172.30.10.3
+          # ...
+    workers:
+      vars:
+        role: worker
+        vendor: Dell
+      hosts:
+        worker1:
+          ansible_host: 192.168.10.16
+          bmc_ip: 172.30.10.6
+          # ...
+        worker2:
+          ansible_host: 192.168.10.17
+          bmc_ip: 172.30.10.7
```

Comment:
Do you think it would be helpful to add sample MAC addresses to the hosts details?

Reply:
Thats part of the `# ...` these are meant to be augmentations to the base not the whole thing. If you want me to add them in 
I can.